src/test/java/org/apache/commons/beanutils2/converters/ConverterTestSuite.java

```diff
@@ -41,7 +41,6 @@ public static Test suite() {
         testSuite.addTestSuite(BooleanConverterTestCase.class);
         testSuite.addTestSuite(ByteConverterTestCase.class);
         testSuite.addTestSuite(CalendarConverterTestCase.class);
-        testSuite.addTestSuite(CharacterConverterTestCase.class);
         testSuite.addTestSuite(ClassConverterTestCase.class);
```

Comment:
Not sure why you deleted this class from this list. What am I missing? Please explain. The class is gone from git master anyway but I am still curious. Please rebase :-)

Reply:
While doing this, I migrated tests of classes I was impacting to JUnit 4. When I changed the `CharacterConverter` I noticed that the line wasn't doing anything, so I removed it assuming it was just some pre-JUnit 4 boilerplate that wasn't needed anymore. ^-^' (Only ever used JUnit 4/5.)